flipt-python/flipt/authentication/__init__.py

```diff
@@ -17,3 +21,43 @@ def __init__(self, token: str) -> None:
 
     def authenticate(self, headers: dict[str, str]) -> None:
         headers["Authorization"] = f"JWT {self.token}"
+
+
+class KubernetesAuthentication(AuthenticationStrategy):
+    def __init__(self,
+                 token: str,
+                 service_account_token_path:str = "/var/run/secrets/kubernetes.io/serviceaccount/token") -> None:
+        self.token = token
+        self.service_account_token_path = service_account_token_path
+        self.token_expiry = None
+
+
+    def authenticate(self, headers: dict[str, str]) -> None:
+        # Check if token is available and not expired.
+        if self.token and self.token_expiry and self.token_expiry > datetime.now().timestamp():
+            headers["Authorization"] = f"Bearer {self.token}"
```

Comment:
should we add `return` here?

Reply:
Yes - Thank you for the spot, I completely missed the return statement.